packages/core/src/utils.ts

```diff
@@ -107,7 +107,9 @@ export async function loadPlugins(context: IContext): Promise<IPluginsLoaded> {
         if (pluginName.startsWith("@octorelease/") && !fs.existsSync(pluginPath)) {
             pluginPath = pluginName.replace("@octorelease", __dirname);
         }
-        pluginsLoaded[pluginName] = require(path.resolve(pluginPath));
+        const fullPluginPath = path.resolve(pluginPath);
+        pluginsLoaded[pluginName] = require(fullPluginPath);
+        Logger.pluginPathMap[pluginName] = require.resolve(fullPluginPath);
```

Comment:
Not a dumb question, initially I used just `fullPluginPath` but that was failing to match plugins in the test run. I found that `fullPluginPath` doesn't include the file extension so `require.resolve` is needed to add it.

Reply:
We may need to enhance this method in the future to support plugins that are not webpacked. For example if the plugin is imported from `samplePlugin/index.ts`, but its stages are defined in `samplePlugin/version.ts` and `samplePlugin/publish.ts`, then we won't be able to trace back to the plugin name, because we are accessing the call stack not the require stack.